speech/google/cloud/speech/client.py

```diff
@@ -139,6 +169,16 @@ def sample(content=None, source_uri=None, encoding=None,
         return Sample(content=content, source_uri=source_uri,
                       encoding=encoding, sample_rate=sample_rate)
 
+    @property
+    def speech_api(self):
+        """Helper for speech-related API calls."""
+        if self._speech_api is None:
+            if self._use_gax:
+                self._speech_api = GAPICSpeechAPI()
```

Comment:
You don't need a `client`? Like will the `Transcript` objects refer to the client that owns them? (I'm assuming no, just want to double check)